src/include/common/types/types.h

```diff
@@ -109,13 +109,14 @@ enum class KUZU_API LogicalTypeID : uint8_t {
 
     STRING = 50,
     BLOB = 51,
-
-    VAR_LIST = 52,
-    STRUCT = 53,
-    MAP = 54,
-    UNION = 55,
-    RDF_VARIANT = 56,
-    POINTER = 57,
+    UUID = 52,
```

Comment:
You can assign typeid 59 to UUID.
Changing the typeid of the existing types may break some tests.